Python/chapter01/1.3 - URLify/lomeli_1.3.py

```diff
@@ -0,0 +1,14 @@
+# Write a method to replace all spaces in a string with '%20'
+# "Mr John Smith     ", 13 => "Mr%20John%20Smith"
+
+def Urlify(string, size):
+    temp = []
+    url = '%20'
+    for i in range(size):
+        if string[i] == ' ':
+            temp.append(url)
+        else:
+            temp.append(string[i])
+    return ''.join(temp)
+
+print(Urlify("Mr John Smith     ", 13))
```

Comment:
It might be better to use the [unittest](https://docs.python.org/3/library/unittest.html) library since that will let you know when a test fails instead of you having to manually go through the output.